changelogs/0.13.md

```diff
@@ -0,0 +1,196 @@
+    - Methods `on_round_started`, `on_fatal`, `on_transaction` to `plugin`
+    - Method `get_round_info` to `application_impl`, `node_delegate`
+    - Method `get_pending_tx_count` to `database`
+    - Method `round_attempts` to `agreement`
+    - Field `_round_attempts` to `echo::rand::config`
+    - Field `_roundinfo` to `agreement`
+    - Method `attempt` to `echo::rand::round`, `echo::rand::step`
+    - Field `attempt` to `block_header`
+    - Field `_attempt` to `block_context_t`, `echo::rand::message`, `echo::rand::round_info`, `echo::rand::step`
+    - Field `_tx_count` to `block_t`
+    - Object `shared_random_t`
+
+## Changed
+- Field `task` in `registration_api` is `optional<registration_task>` now
+- Wallet API
+    - Rename `get_btc_addresses` to [get_btc_addresses](/api-reference/echo-wallet-api/README.md#get_btc_addresses-account)
+- Now frozen balances are taken into account, changed `get_balance` in `interval_tree_selector_impl`
+- Remove empty blocks
+    - Method `broadcast_echorand` return type `size_t` now
+    - Rename method `restart` from echorand `plugin` to `restart_echorand`
+    - Method `start_next_round` from `agreement` return type `void` now
+- Const type max active committee members from `1001` to `19`
+    
+
+## Removed
+- Ethereum sidechain
+    - Field `block_num` in `pending_collect`
+- Field `rewards` in `block_reward_operation`
+- Remove empty blocks
+    - Fields `_last_rand`, `_last_block_hash`, `_last_round` from `echo::rand::options`
+- Field `next_available_vote_id` from `global_property_object`
+- Mechanism increase registration fee
+- Field `maximum_committee_count` from `chain_parameters`
+- Field `committee_member` from `committee_frozen_balance_deposit_operation`
+
+## Fixed
+- Fallback signatures verification
+- Issue block_reward_operation for every account
+- Committee_member_create_operation at zero deposit
+- Bugs in `chain_interface`
+- Incorrect withdraw in Ethereum sidechain
+- Bug with updating ethereum address of a committee member
+- Sidechain called twice in the emulator
+- Signatures check in method `aggregate_btc` from `btc_sidechain`
+- SIGSEGV during messages processing
+- Calling `echorand::on_transaction`
+- Problem with block application because of invalid state of websocket
+- Deprecated-copy error with gcc 9.x
+- Build macOS with unused-parameter as error
+- `Registrer_account` example in wallet-API
+- Fallback message generation
+
+## Improved
+- Updated Ethereum sidechain to reflect the absence of empty blocks
+- Separate `on_maintanance` and `on_committee_update` signals in Database
+- Improved synchronization in echorand plugin
+- Echorand, async dispatching message to active agreement
+- Added limit for x86 contract size in the `create_contract_operation`
+- Handling zero-balances and zero-block at chain start
+- Contract results getters
+- Optimize `interval_tree_selector::get_points`
+
+## Refactoring code with Static Analyzer Code
+
+## Changed data structures
```

Comment:
Probably we should create files with these structures and leave here only a link to files